larsim/EventGenerator/PhotonGen_module.cc

```diff
@@ -110,6 +118,13 @@ namespace evgen {
     , fP{pset.get<double>("P")}
     , fSigmaP{pset.get<double>("SigmaP")}
     , fN{pset.get<int>("N")}
+    , fBminX{pset.get<double>("BminX")}
+    , fBmaxX{pset.get<double>("BmaxX")}
+    , fBminY{pset.get<double>("BminY")}
+    , fBmaxY{pset.get<double>("BmaxY")}
+    , fBminZ{pset.get<double>("BminZ")}
+    , fBmaxZ{pset.get<double>("BmaxZ")}
+
```

Comment:
Two things:

1. I'd rename the configuration parameters (e.g.) `Xmin` instead of `BminX`.
2. Also, this change would require all users to specify `"Bmin*"` and `"Bmax*"` parameters in their configurations, and that is not always desired.  It would be better to make it an option--i.e. if the parameter `Ymin` is provided, then it should override the value of `fYmin`.